File: src/test/kotlin/com/djulia/transactions/JsonHelpers.java
package com.djulia.transactions;

import com.fasterxml.jackson.core.JsonProcessingException;
import com.fasterxml.jackson.databind.ObjectMapper;

import java.math.BigDecimal;

public class JsonHelpers {

    public static String serializeContentForMvcTest(Object value){
        try {
            return new ObjectMapper().writeValueAsString(value);
        } catch (JsonProcessingException e) {
            throw new RuntimeException(e);
        }
    }
}

File: src/test/kotlin/com/djulia/transactions/JsonHelpers.kt
package com.djulia.transactions

import com.fasterxml.jackson.core.JsonProcessingException
import com.fasterxml.jackson.databind.ObjectMapper
import com.fasterxml.jackson.module.kotlin.KotlinModule

import java.math.BigDecimal

object JsonHelpers {

    public val objectMapper = ObjectMapper().registerModule(KotlinModule())

    fun serializeContentForMvcTest(value: Any): String {
        try {
            return objectMapper.writeValueAsString(value)
        } catch (e: JsonProcessingException) {
            throw RuntimeException(e)
        }

    }
}
